test/performance/odp_ipsecfwd.c

```diff
@@ -1119,19 +1169,20 @@ static void drain_events(void)
 static int process_packets(void *args)
 {
 	thread_config_t *config = args;
-	odp_event_t evs[NUM_EVS];
+	odp_event_t evs[MAX_BURST], ev;
 	int cnt;
-	odp_event_t ev;
 	odp_event_type_t type;
 	odp_event_subtype_t subtype;
-	odp_packet_t pkt;
+	odp_packet_t pkt, pkts_in[MAX_BURST], pkts_ips[MAX_BURST];
 	odph_table_t fwd_tbl = config->prog_config->fwd_tbl;
 	stats_t *stats = &config->stats;
 
 	odp_barrier_wait(&config->prog_config->init_barrier);
 
 	while (odp_atomic_load_u32(&is_running)) {
-		cnt = odp_schedule_multi_no_wait(NULL, evs, NUM_EVS);
+		int num_pkts_in = 0, num_pkts_ips = 0;
+
+		cnt = odp_schedule_multi_no_wait(NULL, evs, MAX_BURST);
```

Comment:
The max burst size should be configurable to allow testing without bursts.

Actually, it would be good to have the ipsec enqueue/dequeue burst size configurable separately from scheduling burst size to allow testing limited ipsec burst sizes without affecting scheduling (e.g. to run the kind of test which this test app did before these changes). Maybe that could also be left to a separate patch.

Reply:
This could be a future improvement, added a TODO for it in v2.